test/connection.test.ts

```diff
@@ -523,6 +523,43 @@ describe('Connection', function () {
       assert.calledOnce(spy)
       assert.calledWith(spy, '58')
     })
+
+    it('should not send money in receiveOnly mode', async function ()  {
```

Comment:
Please also add a test to make sure it works as expected when the client is receiveOnly as well